grakn-graql/src/main/java/ai/grakn/graql/internal/gremlin/fragment/Fragment.java

```diff
@@ -99,16 +99,26 @@
     static final double COST_NODE_NOT_INTERNAL = -Math.log(1.1D);
     static final double COST_NODE_IS_ABSTRACT = -Math.log(1.1D);
 
+    public static final double SHARD_LOAD_FACTOR = 0.25 - 1.0; // there is at least one shard
+
+    private static final long DEFAULT_SHARDING_THRESHOLD = 10_000L;
```

Comment:
I guess we can't use the one from the config because that needs an engine dependency :(

Reply:
Do you have access to a transaction/graph/ 🍌  at this stage? If so then i can give you the correct number.

Reply:
Yes please!

Reply:
I will give you a method to get this number then.

Reply:
cool. Thank Filipe!

Reply:
@fppt should I add a commit in this pr or it's something for the future?

Reply:
I would say merge this PR and we can patch it in later.

Reply:
Cool